iina/Extensions.swift

```diff
@@ -330,3 +330,8 @@ extension CharacterSet {
     .union(.urlQueryAllowed)
     .union(.urlFragmentAllowed)
 }
+
+
+extension NSMenuItem {
+  static let dummy = NSMenuItem(title: "Dummy", action: nil, keyEquivalent: "")
```

Comment:
There would be lots of changes since many of them read data from the `NSMenuItem`. If we want to add support for those methods, we may still have to pass dummy `NSMenuItem`s with `tag` or `representedObject` attached.